sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubBufferedProducerAsyncClient.java

```diff
@@ -64,12 +64,23 @@
  * requirement, using {@link EventHubProducerAsyncClient} or {@link EventHubProducerClient} is recommended.
  * </p>
  *
- * <p><strong>Creating an {@link EventHubBufferedProducerAsyncClient}</strong></p>
- * <!-- src_embed com.azure.messaging.eventhubs.eventhubbufferedproducerasyncclient.instantiation -->
+ * <p><strong>Sample: Creating an {@link EventHubBufferedProducerAsyncClient}</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of the asynchronous client
+ * {@link EventHubBufferedProducerAsyncClient}.  The {@code fullyQualifiedNamespace} is the Event Hubs Namespace's host
+ * name. It is listed under the "Essentials" panel after navigating to the Event Hubs Namespace via Azure Portal. The
+ * credential used is {@code DefaultAzureCredential} because it combines commonly used credentials in deployment and
+ * development and chooses the credential to used based on its running environment.  The producer is set to publish
+ * events every 60 seconds with a buffer size of 1500 events for each partition.</p>
+ *
```

Comment:
We can add the same reference for authentication here as package info and remove existing DAC mention there.
```
The examples shown in this document use a credential object named DefaultAzureCredential for authentication, which is appropriate for most scenarios, including local development and production environments. Additionally, we recommend using [managed identity](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/) for authentication in production environments. You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity documentation"](https://learn.microsoft.com/java/api/overview/azure/identity-readme).

```